xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -145,9 +153,10 @@ This constructor is protected. It can only be called from the <xref:System.Runti
         <Parameter Name="isCollectible" Type="System.Boolean" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="isCollectible">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">Value for <see cref="P:System.Runtime.Loader.AssemblyLoadContext.Name" /> in the new instance. Value can be <see langword="null" />.</param>
+        <param name="isCollectible">
+          <see langword="true" /> to enable garbage collection; otherwise, <see langword="false" />.</param>
```

Comment:
I would not use "garbage collection" here - the sentence seems to imply that leaving this false somehow disables GC as a whole.

I would say something like:
> true to enable make the load context collectible, which means it can be unloaded and its memory collected.

Reply:
OK.